requests_auth/authentication.py

```diff
@@ -186,25 +186,50 @@ def __init__(self, token_url: str, username: str, password: str, **kwargs):
             self.data["scope"] = " ".join(scope) if isinstance(scope, list) else scope
         self.data.update(kwargs)
 
+        # As described in https://tools.ietf.org/html/rfc6749#section-6
+        self.refresh_data = {
+            "grant_type": "refresh_token",
+            "refresh_token": None  # the refresh token is set when a refresh is imminent
+        }
+        if scope:
+            self.refresh_data["scope"] = " ".join(scope) if isinstance(scope, list) else scope
```

Comment:
To avoid code duplication :)
```suggestion
        if "scope" in self.data:
            self.refresh_data["scope"] = self.data["scope"]
```